courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2501 @@
+{
```

Comment:
<p>Do the last three lines need to be indented?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<pre class="ql-syntax" spellcheck="false">If the last three lines are indented, we get the SyntaxError: 'return' outside function.   
</pre><p> </p><p> </p><p> </p>